pom.xml

```
@@ -9,11 +9,11 @@
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>signpath</artifactId>
-    <version>${revision}${changelist}</version>
+    <version>${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
         <revision>2.0.1</revision>
-        <changelist></changelist>
+        <changelist>999999-SNAPSHOT</changelist>
         <jenkins.version>2.359</jenkins.version>
```

you shouldn't depend on random weekly versions long term.
the current recommended version is:

```suggestion
        <jenkins.baseline>2.452</jenkins.baseline> 
        <jenkins.version>${jenkins.baseline}.4</jenkins.version>
```

and then update the bom version at the bottom of the file to `<artifactId>bom-${jenkins.baseline}.x</artifactId>`
and version to https://github.com/jenkinsci/bom/releases/tag/3276.vcd71db_867fb_2